pkg/worker/worker.go

```diff
@@ -389,6 +419,33 @@ func (podWorker *PodWorker) Start(ctx context.Context) error {
 			Volumes:        []corev1.Volume{podWorker.storage},
 		},
 	}
+	if podWorker.storage.HostPath != nil {
```

Comment:
Use datasourcetype for this check  ?

Reply:
The podWorker does not have a field for `datasource`, so if use datasource to judge it, we need to add the field.